### DISCUSSION TITLE
Does microcks work with schema referenced from external files ?

### DISCUSSION THREAD

**Discussion by skirankumars31** [microcks/microcks#668] [Q&A] [answered]
Im trying to use a external schema file in my openapi spec file. Everthing works when its all in one file but external file reference does not work for me. Could someone help me figure out what im doing wrong here ?

My files are under the git repo here
https://gitlab.com/kiransukumar/PastryOpenApi/-/tree/feature/ExternalSchemaFile

My open api spec file is https://gitlab.com/kiransukumar/PastryOpenApi/-/blob/feature/ExternalSchemaFile/Pastry.yaml and my schema file is https://gitlab.com/kiransukumar/PastryOpenApi/-/blob/feature/ExternalSchemaFile/PastrySchemas.yaml

Thanks
Kiran

**Comment by lbroudoux** [comment, marked as answer]
This has been covered in #669.
